### DISCUSSION TITLE
Was the ZK Airdrop executed fairly?

### DISCUSSION THREAD

**Discussion by notyourregulardev** [zkSync-Community-Hub/zksync-developers#575] [Polls]
There has been a lot of buzz regarding the eligibility of accounts for the ZK Airdrop. 
Simple question. Do you consider it be accurate and fair distribution?

**Comment by jobair12** [comment]
Please recheck

**Reply by zxuan001** [reply]
**recheck please**

**Reply by faheemraj** [reply]
i have single wallet user i have donete with zksync lite but not count  and multipiler not apply to my address 
this is donete transetion 
https://zkscan.io/explorer/transactions/0x463149cae82fd4a461ba04638891944eb2b3bbd7b6e3f261120c48dc96e3fc01
https://zkscan.io/explorer/transactions/0xc2fb897b2d8f1bc2770c5ad3f12a17bcd66cf9ffbdfc70a27a6b6170ac73f84a
https://zkscan.io/explorer/transactions/0xdf79f3d1f74a81931606ee17c6c9bf426ee21099a05f1ee0081ee8255a92d4d9
2 more trensetion
plz check my address again  and specially multiplayer
0xa2d9f1075fe7f4a054327142b2f5db7fc52d4f87

**Reply by faheemraj** [reply]
i have  only 2011 token not good plz check on Multipliers 6 point but Allocation only 2011

Interacted with 10 smart contracts
Deposited liquidity into DeFi protocols
Traded 10 different ERC20 tokens
Used paymaster for 5 transactions
Transacted on ZKsync Lite in 3 different months before ZKsync Era mainnet

**Reply by danielb-holanda** [reply]
100x difference between min and max allocation is not very community

**Comment by 0xiliyan** [comment]
I got couple PRs:

https://github.com/zksync/credo/pull/60 (19 November 2023)
https://github.com/matter-labs/zksync-web-era-docs/pull/1022 (11 April 2024)

I also developed a dapp on zksync but my Github is not eligible, why? Can you please review guys, we've been supporting the zksync ecosystem.

https://github.com/zkSync-Community-Hub/zksync-developers/discussions/383 (27 February 2023)

@bxpana @0xsisyfos

I'm disappointed that my efforts as a dev who is involved in zkSync ecosystem are not recognized.

**Reply by miloserdow** [reply]
I have a suspicion their checker is lying - I checked all the crypto/zksync contributors I've known and could not find a single eligible person. Were you able to find someone who was eligible?

**Comment by Kixo** [comment]
Recheck please, this is absurd. I am using one wallet. Donated on Gitcoin multiple times even with ZKSync Lite. I have multiple airdrop conditions met and I am not eligible.

**Comment by 0xAditya-eth** [comment]
literally every influencer and their family wallets got max allocation but real users with only 1 wallet got nothing even after owning libertus NFTs, hundreds of transactions. Also the max allocation of 100000 is actually quite high

**Comment by Magadan922** [comment]
Subject: Request for Airdrop Review

Dear zkSync Support Team,

I hope this message finds you well. I am writing to inquire about the recent airdrop. Unfortunately, I did not receive the airdrop, despite having actively contributed to zkSync for a considerable amount of time. I have made numerous transactions and participated in various activities within the zkSync ecosystem.

Could you please review my wallet and check if there was an oversight? My wallet address is [0x973e2b8dBC12DE549213cc83e732A755bFe350d8]. I would greatly appreciate it if you could look into this matter and provide me with any information or assistance.

Thank you for your time and support.

Best regards,

**Comment by danzee1** [comment]
Zksync team really need to recheck. i am not eligible despite

1. Interacting with 500+ Contracts
2. Creating volume of > $10,000
3. Continuously doing transactions each month. More than 9 month consecutively. 
4. Using paymaster & other quests.

**Reply by marcinek89xl** [reply]
Interactions
1404
Volume
$86343
Fee spent
$442.28
Continuously doing transactions each month. More than 15 month consecutively.
0x537b6F941ce307aB8c99a3340Ce137197bE5225F
:(

**Comment by enerelt0418** [comment]
i have only 1 wallet. Omnibus holder. gitcoin more than 20 points. Almost 500$ spend for ecosystem. Still everyday using Era network . But not eligible. 
Interaction 710
Interacted contract 110
Volume 503000$
Fee spend 118$
More than 6 month consecutively. Everyday using Era . 
Please check my wallet: 0xedde1eba17729753aa99bb106f08e1228f4e79c0
https://zk-flow.net/#/wallet/zkSync/0xedDE1EBA17729753Aa99bb106f08E1228f4e79c0

**Reply by karo946** [reply]
Same  0xf40020D82C0a8819B8F542084cB635F764e72769
Volume
$59939
fee spent
$189.20
Activated On
239 days ago
Rank - Minitoolkit
69,378
Active day(s)
132 days
Active week(s)
33 weeks
Active month(s)
10 months

**Comment by Magadan922** [comment]
Could you please review my wallet and check!! My wallet address is [0x973e2b8dBC12DE549213cc83e732A755bFe350d8]. I would greatly appreciate it if you could look into this matter and provide me with any information or assistance.

**Comment by aresstokrat** [comment]
I think they took democracy under their control thats it.
i'm 1.5y+ user, with omnibus nft and github donator in ZkLite - still not eligible.

**Comment by imanpajand** [comment]
No my GitHub account wasn't eligible and my wallet get only 3.5k

**Comment by goldmembertoha** [comment]
40 transactions 15 contracts 15k$ volume + zk lite + memcoin drops for zk activity - not eligible. Zk said that they are for community but they give 30-100k tokens and forget for usual users. One more ghost chain like starknet

**Comment by Reyn35** [comment]
my wallet =
zKlite User
zKsync User
Omibus Claimer
Zigzag user zkLite Claimer
4K+ Vol
242 Trx
86 Contracts
User Old zKsync

why not elig ?

**Reply by Reyn35** [reply]
Pliss cek my wallet : 0x117c89850b3eE64606785dDf2e00182Ac79467ae

**Comment by Lax12** [comment]
Hi Team,
Please check my wallet "0x4bc6efC7D3c6ebe9c7f417B604b4FC27fAAB89Ad"
I have been active on ZKSync network from the beginning and been bridging, interacting with Dapps buying NFT's etc... below are some of the stats of my activity. 
$23,993 volume
$10,952 bridge volume
137 transactions
42 contracts
15 months
Please recheck my account. It would be really disappointing if i am not eligible after investing my time and money.
P.S : my friend who only started  interacting with ZKSync from 3 months got airdrop.

**Comment by shane98x** [comment]
no it wasn't -->
1) too high max allocations
2) sybils ate
3) genuine users got rekt because for one reason lp (even when 20$) had like a 99% weighted factor 
4) all other requirements were placeholders
5) some random external requirements gave 2-5x the amount of actual users with 1 wallet
6) being early was pointless

**Comment by BitcoinPlanets** [comment]
I like the zkSync guys, but that was not a well-planned airdrop from a Sybil perspective... 😞

**Those criteria are easy to miss as a real user and easy to hit as a farmer!** 

It can still be noted that the criteria chosen are significantly higher for many developers and early adopters compared to someone who held a particular NFT a few months ago. Real users might only use 1-2 dapps or a handful of tokens on your L2 when it's so new and has nearly no projects.

I have some developer friends who were certainly among the first when zkSync Lite launched. There wasn't a single decent dApp, so they independently uploaded content to IPFS and minted NFTs. They bridged and exploited all available opportunities, bringing valuable traction to developers in the very very early stages. **Now, they're expected to do that for three months? Holy sh*t.** 

Many of them are getting nothing, 0 $ZK tokens. In contrast, holders of specific airdrops receive an allocation, while the "real users" from the early days are excluded.

This can't be in the interest of ZKNation. On the contrary, it's detrimental to your governance.

**It’s simple really: reward more early users!!!** 

These are the people who used the product because they believed in the vision, not because they were after tokens. These are the users who will hold the token, vote, and help you realize your vision, not your airdrop hunter community.

**Comment by avocadodefi** [comment]
yes it was good, a proper way to remove clusters of industrial farmers. Good job.

**Comment by 0xsisyfos** [comment]
The ZK Token FAQ provides additional details on eligibility criteria: https://docs.zknation.io/zk-token/zk-token-faq

In particular, question 3 ("I met some of the eligibility criteria, but did not get an allocation, why not?") answers your question directly.

**Reply by Kixo** [reply]
nah... not in a long shot. I was using testnet, Lite and Era, deployed several smart contracts. I even proposed to my project manager ZK Sync for our new project. I donated on gitcoin, used several Dexes, deposited in LP. I am NOT eligible. I will NEVER EVER use ZK sync again.
I got OP airdrop, ARB Airdrop and even ZZ airdrop on zksync, but that does not matter, I am still not eligible for unknown reason.
You have a real users and you have farmers and bots. You are rewarding farmers and bots. I dont know who made that decision about 450 ZK as a threshold but you are giving 10k+ ZK to non deserving bots and farmers and not giving anything to the real community. 86% ZK to 5%. GJ

**Comment by FraktalDeFiDAO** [comment]
Please recheck:
https://explorer.zksync.io/address/0x57C173f088254c824aDb26eBDbf3cf6c6f2d0F34#transactions
https://zkscan.io/explorer/accounts/0x57c173f088254c824adb26ebdbf3cf6c6f2d0f34
